source/MaterialXTest/ShaderGen.cpp

```diff
@@ -946,7 +957,7 @@ TEST_CASE("Hello World", "[shadergen]")
         REQUIRE(shader->getSourceCode().length() > 0);
         // Write out to file for inspection
         std::ofstream file;
-        std::string fileName(shader->getName() + "_graph.osl");
+        std::string fileName(RESULT_DIRECTORY + shader->getName() + "_graph.osl");
```

Comment:
Pretty well the rest of the code is pre-pending an output result directory string to output filenames.
All we have left at the top level are the logs.